src/js/BookReader.js

```diff
@@ -302,12 +302,38 @@ BookReader.prototype.initParams = function() {
     }
   }
 
+  /*
+  // If we have a querystring
+  params.queryString = new URLSearchParams(this.readQueryString());
+
+  // Check for Search plugin and term
+  if (this.options.enableSearch) {
```

Comment:
Q: is this meant to be runnable code?

Reply:
Ah. Took me a while to figure out. This was commented out for recent deploy but not committed yet. This would allow q=[term] in standalone BookReader. Putting it back in and testing. Sorry about the confusion.